sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/command/CreateNamespaceSuite.scala

```diff
@@ -25,4 +26,15 @@ import org.apache.spark.sql.execution.command.v1
  */
 class CreateNamespaceSuite extends v1.CreateNamespaceSuiteBase with CommandSuiteBase {
   override def commandVersion: String = super[CreateNamespaceSuiteBase].commandVersion
+
+  test("INVALID_DATABASE_NAME") {
+    val namespace = "ns1.ns2"
+    checkError(
+      exception = intercept[AnalysisException] {
+        sql(s"CREATE NAMESPACE $catalog.$namespace")
```

Comment:
When an user observes the error `The database name is not valid:`, not clear why it is invalid. We should provide some clues.